### DISCUSSION TITLE
Log showing new things like insects and power tools.

### DISCUSSION THREAD

**Discussion by jds3d** [mcguirepr89/BirdNET-Pi#802] [General]
When did the model change to be able to pick up insects and power tools?

I'm seeing all kinds of new things in the log:
'Power tools_Power tools', 0.24983953
'Engine_Engine', 0.2014696
'Amblycorypha longinicta_Common Virtuoso Katydid', 0.31285352
'Orchelimum agile_Agile Meadow Katydid', 0.47526243

**Comment by ehpersonal38** [comment]
Which model are you using?

**Comment by jds3d** [comment]
BirdNET_GLOBAL_3K_V2.2_Model_FP16.  Did it start when I switched to that?

Thanks,
Josh

**Reply by ehpersonal38** [reply]
Yep I think the new model is better at detecting miscellaneous sounds. I think in the labels file you'll even find species like the gray wolf, coyotes, frogs, and crickets. Of course these detections will actually be detected since they're not on our regional species lists.
